docs/source/nested.rst

```diff
@@ -142,14 +142,11 @@ nested_tensor([
 
 Note that nt.unbind()[0] is not a, but rather a slice of the underlying memory, which represents the first entry or constituent of the NestedTensor.
 
-Nested tensor methods
+Nested tensor functions
 +++++++++++++++++++++++++
 
-The following Tensor methods are related to nested tensors:
+The following functions are related to nested tensors:
```

Comment:
"The following functions are specific to nested tensors:" ?

Reply:
Hm, so these will be functions in the nested namespace, hence the change, but they still exist as methods. What do you think would be the best way of putting this across?

Reply:
I think that only functions should be here and methods will be documented on the Tensor object directly?

Reply:
Just to confirm, we still want to_padded_tensor to live as a function in the torch.nested namespace, is that correct?

Reply:
Alban suggested to me that we could potentially remove the method on the tensor base. And use the top level python api to create the method. This python level implementation would under the hood call the torch.nested.to_padded_tensor() function

But yeah I think as long as there is one **function** exposed in python not having both `torch.to_padded_tensor(nt, 0)` and `torch.nested.to_padded_tensor(nt, 0 )` that is good.